CMP1903_A1_2324/Game.cs

```diff
@@ -15,7 +15,31 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
-        //Methods
+        private Die _die1 = new Die(); 
+        private Die _die2 = new Die();
+        private Die _die3 = new Die();
 
+        //created the three die objects 
+
+        private int _roll1 = _die1._roll();
+        private int _roll2 = _die2._roll();
+        private int _roll3 = _die3._roll(); 
+
+        //assigned the roll value to each die
+        //values assigned to private variables
+
+        public int _dieTotal = int _roll1 + int _roll2 + int _roll3;
+
+        //the die values are summed together
+
+        Console.WriteLine($"Die 1 Value: {_roll1}");
+        Console.WriteLine($"Die 2 Value: {_roll2}");
+        Console.WriteLine($"Die 3 Value: {_roll3}");
+        Console.WriteLine($"Total of the three die: {_dieTotal}");
+
+        //outputs the values of each die 
+        //outputs the sum of all three die
```

Comment:
This class works as is stated in the brief and fufills all the requests, pascal casing should be used when naming variables, comments are added for ease of readability and the program works as intended